info.html

```diff
@@ -18,8 +18,57 @@
+          <hr>
+          <center><h2>Community leaders</h2></center>
+          <div class="col-sm-4 leader">
+            <center><img src="" alt="" class="community"></center>
+            <center><h2>Gwen Faraday</h2>
+            <p>Camp Site Leader</p></center>
+          </div>
+          <div class="col-sm-4 leader">
+            <center><img src="" alt="" class="community"></center>
+            <center><h2>Jared Wilcurt</h2>
+            <p>Co-Organizer</p></center>
+          </div>
+          <div class="col-sm-4 leader">
+            <center><img src="" alt="" class="community"></center>
+            <center><h2>Matthew Allbright</h2>
+            <p>Website Project Leader</p></center>
+          </div>
+        </div><!-- end community leaders -->
+        <div class="container">
+          <div class="row">
+            <div class="col-md-12">
+              <center><h1>FreeCodeCamp Code of Conduct</h1></center>
+            </div>
+            <div class="row c-o-c">
+              <div class="col-md6 col-md-offset-1">
+                <p>Free Code Camp is a friendly place to learn to code. We’re committed to keeping it that way.</p>
+                <p>By using Free Code Camp, you agree that you'll follow this code of conduct.</p>
+                <p>In short: Be nice. <b>No harassment, trolling, or spamming.</b></p>
+                <ul>
+                  <li><b>Harassment</b> includes sexual language and imagery, deliberate intimidation, stalking, unwelcome sexual attention, libel and any malicious hacking or social engineering. Free Code Camp should be a harassment-free experience for everyone, regardless of gender, gender identity and expression, age, sexual orientation, disability, physical appearance, body size, race, national origin, or religion (or lack thereof).</li>
+                  <li><b>Trolling</b> includes posting inflammatory comments to provoke an emotional response or disrupt discussions.</li>
+                  <li><b>Spamming</b> includes posting off-topic messages to disrupt discussions, promote a product, or solicit donations.</li>
+                </ul>
+                <p>If you see someone harass, troll, or spam anywhere on Free Code Camp Indy's online media (Discord, YouTube, Facebook, etc.), notify us in via email: <b>freeCodeCampIndy@gmail.com</b> preferably with a screen shot of the offense. Our team will take any action we deem appropriate, up to and including banning the offender from Free Code Camp Indy.</p>
+                <p>Also, no bots are allowed in our chat rooms or on our forum without prior written permission.</p>
+                <p>If you have questions about this code of conduct, email us at <b><a href="mailto:freecodecampindy@gmail.com" target="_top">freeCodeCampIndy@gmail.com</a></b>.</p>
```

Comment:
Replace `<b>`

Use `_blank` for email links instead of `_top` so it opens in a new tab/window instead of a the top most frame (which will just be the current tab).